<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeSettings</class>
 <widget class="QDialog" name="ChangeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="titleLayout">
    <item>
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Portal 2 SDK Manager / Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="titleSeperator">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>431</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="settingsLayout">
    <item>
     <layout class="QVBoxLayout" name="hammerLayout">
      <item>
       <widget class="QLabel" name="hammerLabel">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Hammer - Filename: /hammer(.exe)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hammerBrowseLayout">
        <item>
         <widget class="QLabel" name="hammerBrowseLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hammerBrowse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="facePoserLayout">
      <item>
       <widget class="QLabel" name="facePoserLabel">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Face Poser - Filename: /hlfaceposer(.exe)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="facePoserBrowseLayout">
        <item>
         <widget class="QLabel" name="facePoserBrowseLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="facePoserBrowse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="modelViewerLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Model Viewer - Filename: /hlmv(.exe)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="modelViewerBrowseLayout">
        <item>
         <widget class="QLabel" name="modelViewerBrowseLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="modelViewerBrowse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
